src/components/common/bullets-colin-link.vue

```diff
@@ -91,20 +90,6 @@ export default class BulletsColinLink extends Mixins(NrAffiliationMixin) {
   // For template
   readonly CompanyType = CompanyType
 
-  /**
-   * The alternate codes for entity types.
-   * Alternate codes are used in Entities UIs.
-   */
-  entityTypeAlternateCode (entityType: EntityType): CorpTypeCd {
```

Comment:
Moving this function to Common Mixin. It makes sense since it'll come in handy for even further work later on.